src/main/java/com/json/parser/JsonParser.java

```diff
@@ -0,0 +1,98 @@
+package com.json.parser;
+
+import com.backend.dao.Database;
+import com.fasterxml.jackson.databind.util.JSONPObject;
+import com.utilities.FileReader;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JsonParser {
+
+    public static void main(String args[]) {
+
+        Database database = new Database();
+
+        database.mapDataToMovies("/Users/famin/MoviesDB/simple-movies-database/src/main/resources/static/movies.json");
```

Comment:
Can you think of some way in which we could read the json file on application startup, without explicitly calling `mapDataToMovies`?

Reply:
Added the code into the constructor so that when the class loads, it will load the file in memory